momento-sdk/src/main/java/momento/sdk/responses/topic/SubscriptionState.java

```diff
@@ -0,0 +1,51 @@
+package momento.sdk.responses.topic;
+
+/** Represents the state of a subscription to a topic. */
+public class SubscriptionState {
```

Comment:
I don't think this class needs to be public - we don't expect an end user to ever see it or touch it, right?

Reply:
@nand4011 - I was on a zoom with @rishtigupta and she pointed out that we can't simply remove the 'public' modifier for this, even though it's not supposed to be visible to end users, because it gets used by other classes in our code that do not live in this package.

Do you know of any java paradigms I'm overlooking here that would allow us to prevent this class from being visible in the SDK while still allowing us to reference it from both the Response.Subscription class and our internal Client classes?

Reply:
Not off hand. It's similar to all of the ScsClient and stub manager classes being public. I haven't used modules much, but we may be able to define the public API using one of those.

Reply:
the ScsClient and StubManager classes aren't public though :)

Reply:
Ha, fair enough. I was thinking about them being at the top level instead of in the internal package.